src/energyplus/ForwardTranslator/ForwardTranslateAirLoopHVACUnitarySystem.cpp

```diff
@@ -591,6 +606,14 @@ namespace energyplus {
           } else {
             extensible.setString(UnitarySystemPerformance_MultispeedExtensibleFields::HeatingSpeedSupplyAirFlowRatio, "Autosize");
           }
+        } else if (static_cast<unsigned>(i) < waterToAirHeatingStages.size()) {
```

Comment:
Add support for missing `CoilHeatingWaterToAirHeatPumpVariableSpeedEquationFitSpeedData`.